samples/python2/digits.py

```diff
@@ -74,30 +74,35 @@ def save(self, fn):
 class KNearest(StatModel):
     def __init__(self, k = 3):
         self.k = k
-        self.model = cv2.KNearest()
+        self.model = cv2.ml.KNearest_create()
 
     def train(self, samples, responses):
-        self.model = cv2.KNearest()
-        self.model.train(samples, responses)
+        self.model = cv2.ml.KNearest_create()
+        self.model.train(samples, cv2.ml.ROW_SAMPLE, responses)
 
     def predict(self, samples):
-        retval, results, neigh_resp, dists = self.model.find_nearest(samples, self.k)
+        retval, results, neigh_resp, dists = self.model.findNearest(samples, self.k)
         return results.ravel()
 
 class SVM(StatModel):
     def __init__(self, C = 1, gamma = 0.5):
-        self.params = dict( kernel_type = cv2.SVM_RBF,
-                            svm_type = cv2.SVM_C_SVC,
+        self.params = dict( kernel_type = cv2.ml.SVM_RBF,
```

Comment:
Cheers @berak - i actually realised that after I posted the pull-request. Want me to correct that and apply parameters through the separate methods?